kong/api/routes/config.lua

```diff
@@ -27,21 +28,32 @@ return {
         })
       end
 
-      local entities, err_or_ver
+      local check_hash, old_hash
+      if tostring(self.params.check_hash) == "1" then
```

Comment:
just curious, why are we checking explicitly for `1`? any reason not to check for `true` (meaning no param value needs to be given, just the presence of the param key), or for any non-nil value?

Reply:
From the PR description text: "This design was chosen to allow for future extensibility, if we add new hashing behaviors, those can be set to different numbers." :)

Reply:
Apologies for not opening my eyes :) thanks!